tests/settings/ConfigurationGatheringTest.py

```diff
@@ -177,24 +179,27 @@ def test_back_saving(self):
                                 "SectionManagerTestFile")
 
         # We need to use a bad filename or this will parse coalas .coafile
-        gather_configuration(
-            lambda *args: True,
-            self.log_printer,
-            arg_list=['-S',
-                      "save=" + escape(filename, '\\'),
-                      "-c=some_bad_filename"])
+        # Despite missing settings (coala isn't run) the file is saved
+        with self.assertRaises(SystemExit):
+            gather_configuration(
+                lambda *args: True,
+                self.log_printer,
+                arg_list=['-S',
```

Comment:
Since you choose to break `git blame -w`, I would prefer that you also fix inconsistent quotes, as it is very confusing for newbies to see mixed quotes, and it can be easily fixed while doing other `git blame`-breaking changes.